code/generate_index.py

```diff
@@ -1,18 +1,20 @@
 import os
 import re
+from operator import itemgetter
```

Comment:
I don't think changes from this file should be part of the PR. I think we accidentally picked up changes from a different PR.

Reply:
How do I remove that accidentally added stuff? How can I clean it up? Or can the poluter clean this up, please?